datafusion/functions-nested/src/array_has.rs

```diff
@@ -261,11 +251,13 @@ enum ComparisonType {
     Single,
 }
 
-fn general_array_has_dispatch<O: OffsetSizeTrait>(
+/// Public function for internal benchmark, avoid to use it in production
+pub fn general_array_has_dispatch<O: OffsetSizeTrait>(
```

Comment:
Havn't find any easier way to run private function in benchmark, so make it public

Reply:
My only thought was a wrapper function with a `#[cfg(test)] ` that is public and the benchmark uses that. Slight overhead which likely isn't meaningful here.